npm_and_yarn/lib/dependabot/npm_and_yarn/dependency_files_filterer.rb

```diff
@@ -14,23 +15,28 @@ def initialize(dependency_files:, updated_dependencies:)
       end
 
       def files_requiring_update
-        dependency_files.select do |file|
-          if manifest?(file)
-            package_manifests.include?(file)
-          elsif lockfile?(file)
-            package_manifests.any? do |package_file|
-              File.dirname(package_file.name) == File.dirname(file.name)
+        @files_requiring_update ||=
+          begin
+            dependency_files.select do |file|
+              if manifest?(file)
```

Comment:
I think we want an `if file.support_file?` condition here

Reply:
Oh actually I can probably remove support files as this is now only used to filter manifests and lockfiles for the version resolvers and file updater.